src/api/structure/test.js

```diff
@@ -61,7 +61,7 @@ export default class Test extends TestingUnit {
 
         assertRequestHookType(hooks);
 
-        this.requestHooks = hooks;
+        this.requestHooks = union(this.requestHooks, hooks);
```

Comment:
It changes the method's behavior. Before this change, each call to `test.requestHooks()` overrode the previous value. Now additional calls add additional hooks/scripts.